lib/mocha.js

```diff
@@ -12,6 +12,8 @@ var escapeRe = require('escape-string-regexp');
 var path = require('path');
 var reporters = require('./reporters');
 var utils = require('./utils');
+var Random = require('random-js');
+var MAX_SAFE_INTEGER = 0x1fffffffffffff;
 
```

Comment:
@dasilvacontin Would prefer not to `npm install --save max-safe-integer` (don't know if that exists; probably does) if I can help it.